java/vector/src/main/codegen/templates/UnionVector.java

```diff
@@ -363,12 +361,12 @@ public void copyValueSafe(int from, int to) {
 
   @Override
   public Accessor getAccessor() {
```

Comment:
Where do we stand on these? Remove before 0.8 release?

Reply:
See my comment here to indicate why we have dummy getAccessor() and getMutator() interfaces https://github.com/apache/arrow/pull/1203/files/fb5768fd086a979af8cc8d8795935d191f231678#diff-b5610c173675c3c2707593d7968ee29eR259

Reply:
Sorry the link is not valid any more, can you paste a new link?